src/Stream/src/BinderRabbitMQ/Config/RabbitConsumerOptions.cs

```diff
@@ -54,6 +54,8 @@ public RabbitConsumerOptions()
 
         public string AnonymousGroupPrefix { get; set; }
 
+        public bool? IsEnableBatching { get; set; }
```

Comment:
Does it make sense for this to be nullable? Why not just `bool`?

Reply:
I am just keeping all the options consistent, in this case, I don't know there is a difference.